eng/pipelines/eng-common-sync.yml

```diff
@@ -0,0 +1,35 @@
+# Mirror the eng/common folder to all subscribed langauge repos.
+
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - eng/common
+
+pr: none
+
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: UpdateCreatePullRequestTemplate
```

Comment:
Will need to remove this once my changes go into the azure-sdk-build-tools repo.

Reply:
See https://dev.azure.com/azure-sdk/internal/_git/azure-sdk-build-tools/pullrequest/107